.gitignore

```diff
@@ -36,3 +36,6 @@ docs/vendor/
 
 # Auto-generated build files.
 docker/generated.mk
+
+# Config for loocal running.
```

Comment:
Delete this please. You shouldn't be running experiments from the fuzzbench directory.